src/content/en/2020/resource-hints.md

```diff
@@ -5,19 +5,367 @@ chapter_number: 21
+
+### Best practices
+
+Using Resource Hints can be confusing at times, so let's go over some quick best practices to follow based on Lighthouse's automated audit.
+
+To safely implement `dns-prefetch` and `preconnect` make sure to have them in separate links tags.
+
+```html
+<link rel="preconnect" href="http://example.com">
+<link rel="dns-prefetch" href="http://example.com">
+```
+
+Implementing a `dns-prefetch` fallback in the same `<link>` tag causes a [bug](https://bugs.webkit.org/show_bug.cgi?id=197010) in Safari that cancels the `preconnect` request.
+
+
+{{ figure_markup(
+  caption="Resource hints on desktop that use either preconnect or dns-prefetch use both in the same hint.",
+  content="1.93%",
+  classes="big-number",
+  sheets_gid="281984550",
+  sql_file="preconnect_and_dnsprefetch_relative.sql"
+) }}
+
+Close to 2% of pages (~40k) reported the issue of both `preconnect` & `dns-prefetch` in a single resource.
+
+{{ figure_markup(
+  caption="Pages that pass the preconnect Lighthouse audit",
+  content="19.67%",
+  classes="big-number",
+  sheets_gid="152449420",
+  sql_file="lighthouse_preconnect.sql"
+) }}
+
+We saw only 19.67% of pages passing Lighthouse’s "[Preconnect to required origins](https://web.dev/uses-rel-preconnect/)" audit, creating a large opportunity for thousands of websites to start using `preconnect` or `dns-prefetch` to establish early connections to important third-party origins.
+
+{{ figure_markup(
+  caption="Pages that pass the preload Lighthouse audit",
+  content="84.6%",
+  classes="big-number",
+  sheets_gid="1047875076",
+  sql_file="lighthouse_preload.sql"
+) }}
+
+Running Lighthouse’s "[Preload key requests](https://web.dev/uses-rel-preload/)" audit resulted in 84.6% of pages passing the test, which is an astonishing result. If you are looking to use `preload` for the first time, remember, fonts and critical scripts are a good place to start.
+
+### Native Lazy Loading
+
+Now let’s celebrate the first year of the [Native Lazy Loading](https://addyosmani.com/blog/lazy-loading/) API, which at the time of publishing already has over [72%](https://caniuse.com/loading-lazy-attr) browser support. This new API can be used to defer the load of below-the-fold iframes and images on the page until the user scrolls near them. This can reduce data usage, memory usage, and helps speed up above-the-fold content. Opting-in to lazy load is as simple as adding `loading=lazy`  on `<iframe>` or `<img>` elements.
+
+{{ figure_markup(
+  caption="Percentage of pages using native lazy loading set to “lazy”",
+  content="3.87%",
+  classes="big-number",
+  sheets_gid="2039808014",
+  sql_file="native_lazy_loading_attrs.sql"
+) }}
```

Comment:
Is this images or images and iframes? Think it would be good to clarify (applies to this whole section). And if just images should we have a sentence or two on iframes? We use it on our site :-)